app/src/main/java/org/catroid/catrobat/newui/ui/fragment/LookListFragment.java

```diff
@@ -37,13 +39,18 @@ public static BaseRecyclerListFragment newInstance(int sectionNumber) {
         Bundle args = new Bundle();
         args.putInt(ARG_SECTION_NUMBER, sectionNumber);
         fragment.setArguments(args);
-
         return fragment;
     }
 
     @Override
     public RecyclerViewAdapter<LookInfo> createAdapter() {
-        return new LookAdapter(new ArrayList<LookInfo>(), R.layout.list_item);
+        ArrayList<LookInfo> list = new ArrayList<>();
```

Comment:
looks like something you used for debugging purposes, please remove if that's the case.